src/pages/Meeting/RoomCard/index.jsx

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+import SRoomCard from './style';
+
+export default function RoomCard({
+  localisation,
+  capacity,
+  id,
+  equipment,
+  picture,
+}) {
+  return (
+    <SRoomCard>
+      <h2>Salle de reunion n°{id}</h2>
+      <p>{localisation}</p>
+      <p>Capacité : {capacity} </p>
+      <p>{equipment}</p>
+      <img src={picture} alt="" />
```

Comment:
You forgot the alt here. 
And plz talk with the others to put the same alt langage